inframanager/config.yaml

```diff
@@ -1,16 +1,12 @@
-server:
-  addr: 8080
-
-client:
+GrpcServer:
   addr: localhost:9559
 
-gNMIServer:
+GnmiServer:
   addr: localhost:9339
 
 P4InfoPath: k8s_dp/p4Info.txt
 p4BinPath: k8s_dp/k8s_dp.pb.bin
 DeviceId: 1
-P4ProgConf: k8s_dp/k8s_dp.conf
 HostName: "Node1"
```

Comment:
Where is this Hostname used? If not, can it be removed too?

Reply:
Not used anywhere. Can be removed.